frontend/ios/DroidKaigi 2019/SessionsViewController.swift

```diff
@@ -27,6 +27,18 @@ final class SessionsViewController: UIViewController {
             }
             .disposed(by: disposeBag)
         
+        viewModel.error
+            .bind {[weak self] (errorMessage) in
```

Comment:
Please do not ask me because I do not know iOS. Is it possible to be called more than once, such as when viewDidLoad returns from the background? Will there be errors twice at that time?

Reply:
As far as I know, ``viewDidLoad`` is similar to ``onPostCreate`` in Android.

* [iOS と Android で画面表示時のコールバックを比較する - Qiita](https://qiita.com/amay077/items/52a0b0da97fe455abc08)
* [AndroidエンジニアのためのiOSのUIViewControllerのライフサイクルとAndroidのActivityのライフサイクル比較 - Qiita](https://qiita.com/kgmyshin/items/b54e84d71b3406cd6e79)

So ``viewDidLoad`` does not called when returns from background.